controllers/issues.js

```diff
@@ -0,0 +1,101 @@
+const issuesService = require("../services/issuesService");
+const tasks = require("../models/tasks");
+const { getIssueAssigneeRdsInfo } = require("../models/users");
+const { SOMETHING_WENT_WRONG } = require("../constants/errorMessages");
+
+/**
+ * Get the  issues of the repo
+ * @param {Object} req - Express request object
+ * @param {Object} res - Express response object
+ */
+
+const getIssues = async (req, res) => {
+  try {
+    const issues = await issuesService.getOrgIssues();
+    let issuesData = issues.data.length > 0 ? issues.data : [];
+    issuesData = issuesData.filter((issue) => !Object.keys(issue).includes("pull_request"));
```

Comment:
Is it possible to filter this from GitHub it self?

Reply:
No, by default GitHub considers PRs as issues. There's no query param to filter PRs.